src/components/ChartIcon/ChartIcon.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-const placeholder = require('../placeholder.png');
+const placeholder = require('../../placeholder.png');
 
 interface Props {
   icon: string;
```

Comment:
cleaner to make it `icon?: string` and then do a defined check later:
```
const iconSrc = icon || `/api/chartsvc/${icon}`
```

Reply:
:+1: thanks, will do this in a separate PR